src/SprykerEco/Zed/Unzer/Business/UnzerFacadeInterface.php

```diff
@@ -44,11 +46,10 @@ public function performPreSaveOrderStack(QuoteTransfer $quoteTransfer): QuoteTra
 
     /**
      * Specification:
-     * - Requires `QuoteTransfer::payment` to be set.
-     * - Requires `PaymentTransfer::unzerPayment` to be set.
-     * - Requires `UnzerPaymentTransfer::unzerCustomer` to be set.
-     * - Requires `UnzerPaymentTransfer::unzerKeypair` to be set.
-     * - Requires `UnzerKeypairTransfer::keypairId` to be set.
+     * - Requires `QuoteTransfer.payment.unzerPayment.idSalesOrder` to be set.
+     * - Requires `QuoteTransfer.payment.unzerPayment.unzerCustomer` to be set.
+     * - Requires `QuoteTransfer.payment.unzerPayment.unzerKeypair.keypairId` to be set.
+     * - Requires `SaveOrderTransfer.orderReference` to bet set.
```

Comment:
I don't see where set `IdPaymentUnzer` property, but than it used like `$paymentUnzerOrderItemTransfer->getIdPaymentUnzerOrFail()`

Reply:
resolved in mapper